src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -545,6 +545,13 @@ public static class PublishedContentExtensions
                 // Simple types
                 result = propertyValue;
             }
+            else if (propertyValue is IPublishedContent)
```

Comment:
(from Skype) @mattbrailsford suggests that we change to:

```csharp
else if (propertyValue is IPublishedContent && propertyInfo.PropertyType.IsClass)
```

*making a note here so I don't forget later*